packages/lab/src/layout/LayerLayout/LayerLayout.tsx

```diff
@@ -1,12 +1,12 @@
 import { forwardRef, HTMLAttributes, useState, useEffect } from "react";
 import cx from "classnames";
 
+import { Scrim, ScrimProps } from "@jpmorganchase/uitk-core";
```

Comment:
These imports should merging

Reply:
You mean, should not be merged?

Reply:
Should I do the following?

```ts
import { Scrim } from "@jpmorganchase/uitk-core";
import type { ScrimProps } from "@jpmorganchase/uitk-core";
```

Reply:
Sorry, as in with the other imports from the core package in the file.

Reply:
Ah, right. Im blind :'D